jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBCCore.java

```diff
@@ -192,17 +192,38 @@ public void executeQuery(ExtensionServiceMessage message) {
                         "JDBC connection closed before operation could complete.", null);
             }
         }
-        
+
+        // Check if query request is actually an update
+        boolean isUpdate = false;
+        if (request.get("isUpdate") instanceof Boolean && (Boolean) request.get("isUpdate")) {
+            isUpdate = (Boolean) request.get("isUpdate");
+        }
+
```

Comment:
Just set `isUpdate` here by querying the query?  Then no other code must change?